### ISSUE TITLE
Comparison of changes in fecal microbiota of calves with and without dam

### ISSUE THREAD

**Issue by Omabekee** [waldronlab/BugSigDBcuration#151]
Comparison of changes in fecal microbiota of calves with and without dam -- Li M et al. -- PeerJ

https://peerj.com/articles/12826/

**Comment by Folakunmi21** [comment]
I'll like to work on this @cmirzayi

**Comment by lwaldron** [comment]
@Folakunmi21 please let us know if you still want to curate this even though it's no longer during the Outreachy contribution period.

**Comment by Folakunmi21** [comment]
@lwaldron yes sir. I've curated this but I need to comment on some of the signatures before I can submit it for review. I don't have reviewers permissions that'll enable me to make the comments.

**Comment by lwaldron** [comment]
@Folakunmi21 I just assigned Reviewer permissions to you, so you can leave comments on taxa now.

**Comment by Folakunmi21** [comment]
@lwaldron Thank you. Kindly review the curation here: https://bugsigdb.org/Study_841

**Comment by Peacesandy** [comment]
Reviewed
https://bugsigdb.org/Study_841

**Comment by Peacesandy** [comment]
Reviewed
https://bugsigdb.org/Study_841
